Source/Resources/Content/MSGraph/ConditionalAccess/Policies/Require Approved Client Apps from Unmanaged Devices and Untrusted Locations.json

```diff
@@ -55,7 +55,7 @@
       ]
     }
   },
-  "description": "Allows only content protected and whitelisted mobile (iOS and Android) apps from off-network.",
+  "description": "What is it: Allows only allowlisted mobile (iOS and Android) apps to connect from off-network. \n Simeon Notes: This policy helps protect your data. Application protection policies on iOS and Android have been tested and verified to protect data, these protection policies do not work from other device types. Intune managed devices are still able to access data off network. Simeon's baseline assigns the policy to all users except those in the Azure AD group 'BUILTIN - Unrestricted Access From Unmanaged Devices And Untrusted Locations'. The policy is also assigned to all applications except those listed in the config property 'AppsAllowedFromUntrustedLocations'.\n Learn More: https://docs.microsoft.com/en-us/mem/intune/apps/app-protection-policy",
```

Comment:
**What does this do?**
Allows only 'allowlisted' mobile (iOS and Android) apps to connect from off-network. The baseline assigns the policy to all users except those in the Azure AD group 'BUILTIN - Unrestricted Access From Unmanaged Devices And Untrusted Locations'. The policy is also assigned to all applications except those listed in the config property 'AppsAllowedFromUntrustedLocations'. These protection policies do not work from other device types. Intune-managed devices are still able to access data off network. <-- @metamenos 'from' or 'for' in the last sentence?

**Why should you use this?**
This policy helps protect your data, as application protection policies on iOS and Android have been tested and verified to protect data. 

**What is the end-user impact?**
Users may only connect from off-network via iOS and Android devices. 

**Learn more**
[How you can protect app data](https://docs.microsoft.com/en-us/mem/intune/apps/app-protection-policy)